src/buildercore/cfngen.py

```diff
@@ -154,6 +167,11 @@ def write_template(stackname, contents):
     open(output_fname, 'w').write(contents)
     return output_fname
 
+def read_template(stackname):
+    output_fname = os.path.join(STACK_DIR, stackname + ".json")
+    with open(output_fname, 'r') as template_file:
```

Comment:
there is a more straightforward: `return json.load(open(output_fname), 'r'))`

the `s` in `loads` and `dumps` is for `string`